cmd/policy-assistant/examples/demos/walkthrough/README.md

```diff
@@ -0,0 +1,115 @@
+|         |    demo    |                                    |                       |    Allow any peers              |                          |
+|         | Pod:       |                                    |                       |                                 |                          |
+|         |    pod = a |                                    |                       |                                 |                          |
++         +------------+------------------------------------+-----------------------+---------------------------------+--------------------------+
+|         | Namespace: | [ANP] default/anp1                 | Namespace:            | BANP:                           | all ports, all protocols |
+|         |    all     | [ANP] default/anp2                 |    all                |    Deny                         |                          |
+|         |            | [BANP] default/default             | Pod:                  |                                 |                          |
+|         |            |                                    |    all                |                                 |                          |
++         +            +                                    +-----------------------+---------------------------------+                          +
+|         |            |                                    | Namespace:            | ANP:                            |                          |
+|         |            |                                    |    development = true |    pri=2 (development-ns): Pass |                          |
+|         |            |                                    | Pod:                  |                                 |                          |
+|         |            |                                    |    all                |                                 |                          |
++         +            +                                    +-----------------------+---------------------------------+--------------------------+
+|         |            |                                    | Namespace:            | ANP:                            | port 80 on protocol TCP  |
+|         |            |                                    |    all                |    pri=1 (allow-80): Allow      |                          |
+|         |            |                                    | Pod:                  |                                 |                          |
+|         |            |                                    |    all                |                                 |                          |
++---------+------------+------------------------------------+-----------------------+---------------------------------+--------------------------+
+```
+
+## Demo
+
+To try for yourself:
+
+1. Download `policy-assistant` via the [Quick Install](../../../README.md#quick-install) guide.
+1. Leverage the JSON/YAML files in this folder.
```

Comment:
nit: bullet points here are all 1?
also how exactly would they leverage these yamls/json.. ? is it by providing them in the path ?

Reply:
https://github.com/kubernetes-sigs/network-policy-api/pull/269#issuecomment-2475321625

also md let's you specify 1's instead of using 1,2,3, 😜